src/ruby/.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
Local configuration file for RSpec. There is also a global configuration file in `~/`dir. The local overrides the global one.